docs/experiments-doc2query.md

```diff
@@ -10,7 +10,7 @@ Here, we run through how to replicate the BM25+Doc2query condition with our copy
 ## MS MARCO Passage Ranking
 
 To replicate our Doc2query results on the [MS MARCO Passage Ranking Task](https://github.com/microsoft/MSMARCO-Passage-Ranking), follow these instructions.
-Before going through this guide, it might make sense to [replicate our BM25 baselines](experiments-msmarco-passage.md) first.
+Before going through this guide, it is recommended to [replicate our BM25 baselines](experiments-msmarco-passage.md) first.
```

Comment:
I don't think this is grammatical? Maybe "It is recommended that you replicate..."?